PSICover/PSICover.cs

```diff
@@ -163,11 +166,21 @@ class Analyzer {
          var code = File.ReadAllLines (file);
          for (int i = 0; i < code.Length; i++)
             code[i] = code[i].Replace ('<', '\u00ab').Replace ('>', '\u00bb');
+         int hitCount = 0;

```

Comment:
Metamation coding convention variable should start with c for counts. So here it should be cHits